docs/reference/commandline/network_ls.md

```diff
@@ -12,7 +12,7 @@ parent = "smn_cli"
 
     Usage:  docker network ls [OPTIONS]
 
-    Lists all the networks created by the user
+    List networks
```

Comment:
hm, still something we need to decide on it think, for consistency (`list` vs `lists`)